gtfs/spec/en/reference.md

```diff
@@ -209,244 +211,183 @@ The example below is valid, with distinct blocks every day of the week.
-|  end_time | Time | **Required** | The **end_time** field indicates the time at which service changes to a different frequency (or ceases) at the first stop in the trip. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service day. For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. E.g. 25:35:00. |
-|  headway_secs | Non-negative integer | **Required** | The **headway_secs** field indicates the time between departures from the same stop (headway) for this trip type, during the time interval specified by **start_time** and **end_time**. The headway value must be entered in seconds. |
-|   |  | | Periods in which headways are defined (the rows in frequencies.txt) shouldn't overlap for the same trip, since it's hard to determine what should be inferred from two overlapping headways. However, a headway period may begin at the exact same time that another one ends, for instance: |
-|   |  | | `A, 05:00:00, 07:00:00, 600` |
-|   |  | | `B, 07:00:00, 12:00:00, 1200` |
-|  exact_times | Enum | Optional | The **exact_times** field determines if frequency-based trips should be exactly scheduled based on the specified headway information. Valid values for this field are: |
-|   |  | | * **0** or **(empty)** - Frequency-based trips are not exactly scheduled. This is the default behavior. |
-|   |  | | * **1** - Frequency-based trips are exactly scheduled. For a frequencies.txt row, trips are scheduled starting with trip_start_time = start_time + x * headway_secs for all x in (0, 1, 2, ...) where trip_start_time < end_time. |
-|   |  | | The value of **exact_times** must be the same for all frequencies.txt rows with the same **trip_id**. If **exact_times** is 1 and a frequencies.txt row has a **start_time** equal to **end_time**, no trip must be scheduled. When **exact_times** is 1, care must be taken to choose an **end_time** value that is greater than the last desired trip start time but less than the last desired trip start time + **headway_secs**. |
+|  trip_id | ID referencing `trips.trip_id` | **Required** | Identifies a trip to which the specified frequency of service applies. |
+|  start_time | Time | **Required** | Time at which the first vehicle departs from the first stop of the trip with the specified frequency. |
+|  end_time | Time | **Required** | Time at which service changes to a different frequency (or ceases) at the first stop in the trip. |
```

Comment:
Change: "Time at which service changes to a different headway (or ceases) at the first stop in the trip."

Justification: "frequency" is indeed a term used for the file name, but the spec refers to headways, which is a more familiar transit term.